src/Utilities/ConstantExpressions.hpp

```diff
@@ -51,7 +51,7 @@ SPECTRE_ALWAYS_INLINE constexpr size_t get_nth_bit(const size_t i,
 /// \ingroup ConstantExpressionsGroup
 /// \brief Compute the square of `x`
 template <typename T>
-SPECTRE_ALWAYS_INLINE constexpr decltype(auto) square(const T& x) {
+SPECTRE_ALWAYS_INLINE constexpr decltype(auto) square(const T& x) {  // NOLINT
```

Comment:
what is NOLINTed here?

Reply:
This one was `const-return-type` though now I think it's globally disabled so I'll just remove this `NOLINT`